app/src/main/java/org/cru/godtools/fragment/ToolDetailsFragment.java

```diff
@@ -288,6 +295,17 @@ private void updateViews() {
         }
         bindShares(mShares, mTool);
 
+        boolean hasOverviewVideo = mVideoBanner != null && mTool != null &&
+                !TextUtils.isEmpty(mTool.getOverviewVideo());
+        if (mBanner != null) {
+            mBanner.asImageView().setVisibility(hasOverviewVideo ? View.GONE : View.VISIBLE);
+        }
+        if (mVideoBanner != null) {
+            mVideoBanner.setVisibility(hasOverviewVideo ? View.VISIBLE : View.GONE);
+            if (hasOverviewVideo) {
+                updateOverviewVideo();
```

Comment:
what happens if this function is triggered while a video is already playing?

the `updateViews()` method can be triggered multiple times as data is loaded/reloaded.